Formula/e/esbuild.rb

```diff
@@ -17,11 +15,12 @@ class Esbuild < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "cf5ea30dc768145e289400d5e95ef9cfd467641b9ce06781334cda6360d59969"
   end
 
-  depends_on "node"
+  depends_on "go" => :build
+  depends_on "node" => :test
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
```suggestion
    ENV["CGO_ENABLED"] = OS.mac? ? "1" : "0"
```

Reply:
why? I'm following upstream here: https://github.com/evanw/esbuild/blob/360d47230813e67d0312ad754cad2b6ee09b151b/Makefile#L10

Reply:
I think it would be good to have CGO enabled, upstream is doing cross build that is why they do CGO enabled false. 

for our linux builds, we cannot do due to https://github.com/Homebrew/homebrew-core/issues/163826

Reply:
gotcha, thanks for explaining :)